### PR TITLE
Aboutpageheader

### PR DESCRIPTION
**Pull request by marwaghassa**
This resolve issue #10

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,11 +3,12 @@ import React from "react";
 import Footer from "./components/Footer/Footer";
 import { BrowserRouter as Router } from "react-router-dom";
 import Navbar from "./components/Navbar/Navbar";
-
+import Aboutpageheader from "./components/aboutpageheader/aboutPageHeader";
 function App() {
   return (
     <Router>
       <Navbar />
+      <Aboutpageheader />
```
Comment (osamaakb) [inline src/App.js:11]: what did we say about not changing the `app.js`?

File: src/components/aboutpageheader/aboutpageheader.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Leaf1 from "../../images/about";
+function Aboutpageheader() {
```
Comment (osamaakb) [inline src/components/aboutpageheader/aboutpageheader.jsx:3]: the component, file and folder names aren't correct please fix them it should be `AboutPageHeader` for all of them

File: tailwind.config.js [hunk 3]
```diff
@@ -17,6 +17,8 @@ module.exports = {
       gray: "#c5cbc5",
       dark: "#717771",
       tan: "#fcfbf9",
+      about:"#892E2E",
```
Comment (osamaakb) [inline tailwind.config.js:20]: this color name doesn't make sense

**Review by osamaakb — CHANGES_REQUESTED** [review]
you have a lot of image files, what are they about? you only need one image

**Comment by marwaghassa** [comment]
Requested changes done .

**Review by osamaakb — APPROVED** [review]

**Pull request merged**